tests/fixtures/Page.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\CMS\Controllers\ContentController;
+
+class Page extends SiteTree
```

Comment:
The copy command is manual in `.travis.yml`; the travis test run will never make use of a mysite folder. It's kept in a manifest-excluded folder to prevent conflicts with other code.

I could perform the copy in bootstrap.php instead of travis.yml, although detecting if $project is set will get a chicken-egg bug: you'd need the manifest to be built to check that value, and the file needs to be copied before the manifest is built.